<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XAFS</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XAFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Energy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Energy step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Integral time(s)</string>
   </property>
  </widget>
  <widget class="PyDMScatterPlot" name="PyDMScatterPlot">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>
    PyDMScatterPlot is a widget to plot one scalar value against another.
    Multiple scalar pairs can be plotted on the same plot.  Each pair has
    a buffer which stores previous values.  All values in the buffer are
    drawn.  The buffer size for each pair is user configurable.

    Parameters
    ----------
    parent : optional
        The parent of this widget.
    init_x_channels: optional
        init_x_channels can be a string with the address for a channel,
        or a list of strings, each containing an address for a channel.
        If not specified, y-axis waveforms will be plotted against their
        indices.  If a list is specified for both init_x_channels and
        init_y_channels, they both must have the same length.
        If a single x channel was specified, and a list of y channels are
        specified, all y channels will be plotted against the same x channel.
    init_y_channels: optional
        init_y_channels can be a string with the address for a channel,
        or a list of strings, each containing an address for a channel.
        If a list is specified for both init_x_channels and init_y_channels,
        they both must have the same length.
        If a single x channel was specified, and a list of y channels are
        specified, all y channels will be plotted against the same x channel.
    background: optional
        The background color for the plot. Accepts any arguments that
        pyqtgraph.mkColor will accept.
    </string>
   </property>
   <property name="showXGrid">
    <bool>true</bool>
   </property>
   <property name="showYGrid">
    <bool>true</bool>
   </property>
   <property name="backgroundColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="axisColor">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="curves">
    <stringlist>
     <string>{&quot;y_channel&quot;: &quot;m1.DRBV&quot;, &quot;x_channel&quot;: &quot;m1.DRBV&quot;, &quot;name&quot;: &quot;&quot;, &quot;color&quot;: &quot;red&quot;, &quot;lineStyle&quot;: 0, &quot;lineWidth&quot;: 2, &quot;symbol&quot;: &quot;o&quot;, &quot;symbolSize&quot;: 5, &quot;redraw_mode&quot;: 0, &quot;buffer_size&quot;: 1200}</string>
    </stringlist>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I0</string>
   </property>
  </widget>
  <widget class="PyDMScatterPlot" name="PyDMScatterPlot_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>
    PyDMScatterPlot is a widget to plot one scalar value against another.
    Multiple scalar pairs can be plotted on the same plot.  Each pair has
    a buffer which stores previous values.  All values in the buffer are
    drawn.  The buffer size for each pair is user configurable.

    Parameters
    ----------
    parent : optional
        The parent of this widget.
    init_x_channels: optional
        init_x_channels can be a string with the address for a channel,
        or a list of strings, each containing an address for a channel.
        If not specified, y-axis waveforms will be plotted against their
        indices.  If a list is specified for both init_x_channels and
        init_y_channels, they both must have the same length.
        If a single x channel was specified, and a list of y channels are
        specified, all y channels will be plotted against the same x channel.
    init_y_channels: optional
        init_y_channels can be a string with the address for a channel,
        or a list of strings, each containing an address for a channel.
        If a list is specified for both init_x_channels and init_y_channels,
        they both must have the same length.
        If a single x channel was specified, and a list of y channels are
        specified, all y channels will be plotted against the same x channel.
    background: optional
        The background color for the plot. Accepts any arguments that
        pyqtgraph.mkColor will accept.
    </string>
   </property>
   <property name="showXGrid">
    <bool>true</bool>
   </property>
   <property name="showYGrid">
    <bool>true</bool>
   </property>
   <property name="backgroundColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="axisColor">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="curves">
    <stringlist>
     <string>{&quot;y_channel&quot;: &quot;m1.DRBV&quot;, &quot;x_channel&quot;: &quot;m1.DRBV&quot;, &quot;name&quot;: &quot;&quot;, &quot;color&quot;: &quot;red&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 2, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;redraw_mode&quot;: 0, &quot;buffer_size&quot;: 1200}</string>
    </stringlist>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>settling time(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>next energy</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy0</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy1</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy3</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy4</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy7</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy5</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_10">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy6</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step5</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step2</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step4</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step3</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_15">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step6</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_16">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step0</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_17">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Step1</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_18">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime5</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_19">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime1</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_20">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime3</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_21">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime2</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_22">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime4</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_23">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime6</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_24">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:ITime0</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Next</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_25">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:SettlingTime</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:XAFS:Energy2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPara">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>parameters</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>m1.DRBV</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>417:aiExample</string>
   </property>
   <property name="displayFormat" stdset="0">
    <enum>PyDMLabel::Default</enum>
   </property>
  </widget>
  <widget class="PyDMScatterPlot" name="PyDMScatterPlot_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>531</width>
     <height>391</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>
    PyDMScatterPlot is a widget to plot one scalar value against another.
    Multiple scalar pairs can be plotted on the same plot.  Each pair has
    a buffer which stores previous values.  All values in the buffer are
    drawn.  The buffer size for each pair is user configurable.

    Parameters
    ----------
    parent : optional
        The parent of this widget.
    init_x_channels: optional
        init_x_channels can be a string with the address for a channel,
        or a list of strings, each containing an address for a channel.
        If not specified, y-axis waveforms will be plotted against their
        indices.  If a list is specified for both init_x_channels and
        init_y_channels, they both must have the same length.
        If a single x channel was specified, and a list of y channels are
        specified, all y channels will be plotted against the same x channel.
    init_y_channels: optional
        init_y_channels can be a string with the address for a channel,
        or a list of strings, each containing an address for a channel.
        If a list is specified for both init_x_channels and init_y_channels,
        they both must have the same length.
        If a single x channel was specified, and a list of y channels are
        specified, all y channels will be plotted against the same x channel.
    background: optional
        The background color for the plot. Accepts any arguments that
        pyqtgraph.mkColor will accept.
    </string>
   </property>
   <property name="showXGrid">
    <bool>true</bool>
   </property>
   <property name="showYGrid">
    <bool>true</bool>
   </property>
   <property name="backgroundColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="axisColor">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="curves">
    <stringlist/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ln(I0/I1)</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMScatterPlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.scatterplot</header>
  </customwidget>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
